plat/arm/common/aarch64/arm_common.c

```diff
@@ -150,6 +151,7 @@ void arm_configure_sys_timer(void)
 
 	reg_val = (1 << CNTNSAR_NS_SHIFT(PLAT_ARM_NSTIMER_FRAME_ID));
 	mmio_write_32(ARM_SYS_TIMCTL_BASE + CNTNSAR, reg_val);
+#endif
```

Comment:
Ah yes, I remember now we discussed this before.

It might be slightly safer to wrap the whole function, not just the scoped contents. That way, a link error will be raised if a platform tries to use `arm_configure_sys_timer()` without defining `ARM_SYS_TIMCTL_BASE`.